digitalearthau/qsub.py

```diff
@@ -67,6 +67,7 @@ def __call__(self, *args, **kwargs):
             args = sys.argv[1:]
             args = remove_args('--qsub', args, n=1)
             args = remove_args('--queue-size', args, n=1)
+            args = remove_args('--batch', args, n=1)
```

Comment:
This is getting uglier and uglier. When it was just `--qsub` it was okish, but now the list keeps growing and it has nothing to do with this class. This logic needs to be moved up to the caller, whoever adds internal args should also supply auto-clean list of args, or maybe caller to this function should supply it

```python
auto_clean = [ ('--batch', 1), '-v', ('--verbose',0)] 

for x in auto_clean:
    arg, n = (x,0) if isininstance(x, str) else x
    args = remove_args(arg, args, n=n)
```

Reply:
But I want the verbose flag to propagate 😕

Reply:
I mean it as an example of input to the function, it shouldn't be defined in here, but passed in by the caller, since the set of arguments to be pruned should be caller responsibility